### DISCUSSION TITLE
Mac 中文化問題

### DISCUSSION THREAD

**Discussion by Nokcc** [GpointChen/FFXIVChnTextPatch-GP#293] [Q&A]
你好 我沒有Coding 知識
嘗試在mac 中文化在最後 gradle run 的時候
彈出了以下信息

goolgle 查了很多也不太明白如何做
請問有方法可以解決嗎？

* Where:
Initialization script '/Users/xyz/Downloads/gradle-6.8.3/init.d/build.gradle' line: 18

* What went wrong:
A problem occurred evaluating initialization script.
> Failed to apply plugin 'org.gradle.idea'.
   > Could not create plugin of type 'IdeaPlugin'.
      > Unable to determine constructor argument #3: missing parameter of type IdeArtifactRegistry, or no service of type IdeArtifactRegistry.

**Comment by Nokcc** [comment]
另外FFXIVChnTextPatch-1222 file 
請問是應該放在download>gradle-6.8.3>裡頭嗎，在運行gradle run 就會執行安裝檔嗎？
這個部分有點不太明白

**Comment by GpointChen** [comment]
我個人沒在mac類環境操作過，不確定你遇到了什麼問題，可能要看有沒有其他人了解。

**Comment by hydai** [comment]
Hi @Nokcc 
我有在 macos 上面操作成功過，基本上過程跟教學裡的差不多，根據你的錯誤 `/Users/xyz/Downloads/gradle-6.8.3/init.d/build.gradle` ，這個路徑應該要改成你下載並解壓縮後的 gradle 路徑
